bingosync-app/generators/dark_souls_generator.js

```diff
@@ -201,17 +198,17 @@ bingoList[20] = [
   { name: "Obtain Painting Guardian Set", types: ["paintedworld","armour"] },
   { name: "Obtain Velka's Rapier", types: ["paintedworld","weapons"] },
   { name: "Murder Priscilla", types: ["paintedworld","youmonster"] },
-  { name: "Obtain Dried Finger", types: ["paintedworld","items"] }
+  { name: "Obtain a Gold Coin", types: ["paintedworld","items"] }
```

Comment:
Replaced "Obtain Dried Finger" with "Obtain a Gold Coin", as there is a **big** discrepancy in difficulty of obtaining Dried Finger, depending on whether the player is playing Dark Souls Remastered. Acquiring Dried Finger is trivial in the remaster.

The Dried Finger challenge is intended as a Painted World of Ariamis challenge. Obtaining a Gold Coin seems like an interesting replacement as one can be obtained in the Painted World of Ariamis, but the player has three options for where to get one, with the other two being of similar difficulty and require a similar level of completion.